types/src/v0/impls/solver.rs

```diff
@@ -0,0 +1,67 @@
+use committable::{Commitment, Committable};
+use hotshot::types::SignatureKey;
+
+use super::v0_3::{RollupRegistrationBody, RollupUpdatebody};
+
+impl Committable for RollupRegistrationBody {
```

Comment:
I'm wondering if it would be better to do the implementation on `RollupRegistration`, since that is the thing that will be getting passed around. `commit()` could just ignore the signature.

Reply:
So the signature is created by signing the commit of registration body. If the commit() is implemented over the the outer struct then I would need to insert a random signature -> sign the commit() and then overwrite the signature or am I missing something?

Reply:
Yea, silly me.